CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.8.0+education-1.0.0] - 2020-03-30
```

Comment:
how do you intend to manage the +education variant if you don't have 2 branches?

Reply:
I think we can handle it in the same master branch with tags.